packages/wdio-crossbrowsertesting-service/src/service.js

```diff
@@ -146,23 +146,17 @@ export default class CrossBrowserTestingService {
         return this.updateJob(oldSessionId, this.failures, true, browserName)
     }
 
-    updateJob (sessionId, failures, calledOnReload = false, browserName) {
-        return new Promise((resolve, reject) => request.put(this.getRestUrl(sessionId), {
-            json: true,
-            auth: {
-                user: this.cbtUsername,
-                pass: this.cbtAuthkey
-            },
-            body: this.getBody(failures, calledOnReload, browserName)
-        }, (e, res, body) => {
-            /* istanbul ignore if */
-            this.failures = 0
-            if (e) {
-                return reject(e)
-            }
-            global.browser.jobData = body
-            return resolve(body)
-        }))
+    async updateJob (sessionId, failures, calledOnReload = false, browserName) {
+        const json = this.getBody(failures, calledOnReload, browserName)
+        this.failures = 0
+        const response = await got.put(this.getRestUrl(sessionId), {
+            json,
```

Comment:
shouldn't it be `json: true, body: json` ?

Reply:
can't understand how it works.
I worked with `got` in my PR and have `body` everywhere instead of `json` that you have as `body`, while `json` is a boolean flag https://www.npmjs.com/package/got#json

Reply:
> shouldn't it be `json: true, body: json` ?

I got confused with the `request` lib here. Will fix this.

Reply:
> shouldn't it be `json: true, body: json` ?

As far as I understand this: `json` should be used if you have json payload and `body` if you want to send other types e.g. strings. I don't see a reason why we should use `body`.